components/ImageDisplay.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <div class="image-container" @click.prevent.left.exact="copy(false)" @click.prevent.exact.right="copy(true)" @click.prevent.shift="favorite()">
```

Comment:
We want to keep the original functionality in place for desktop users, this should stay.